cypress/util.js

```diff
@@ -0,0 +1,9 @@
+export const baseURL = 'http://localhost:4200/';
```

Comment:
This is no longer necessary and can be removed.

Reply:
Done